CMakeLists.txt

```diff
@@ -77,6 +77,8 @@ else()
     set(ARCH s390x)
   elseif(CMAKE_SYSTEM_PROCESSOR MATCHES "^(sparc.*|SPARC.*)")
     set(ARCH sparc)
+  elseif(CMAKE_SYSTEM_PROCESSOR MATCHES "^(loongarch.*|LOONGARCH.*)")
```

Comment:
Has the while repo been validated on this ARCH?

Reply:
For whole repo, I meant to ask the features and build flags to the pasted test result above.

Reply:
> Has the while repo been validated on this ARCH? If yes, probably we can add a CI workflow for it?

I apologize for the other items in the repository that I haven't tested yet

Reply:
Ok. I think it is good to merge it based on the current test results.